web/app/views/applications/new.html.erb

```diff
@@ -12,6 +12,7 @@
             prevented this application from being created.
         </div>
         <% end %>
+        <%= f.hidden_field :user_id, value: @form.user_id %>
```

Comment:
This is ok for now but before long, we should figure out some way to pass auth credentials to the API, because otherwise a malicious user could just pass any user ID here. Can you open a separate issue to track that?

Reply:
Agreed, I'll open an issue for that.